cms/djangoapps/contentstore/views/item.py

```diff
@@ -631,10 +665,77 @@ def _create_item(request):
     )
 
     return JsonResponse(
-        {"locator": unicode(created_block.location), "courseKey": unicode(created_block.location.course_key)}
+        {'locator': unicode(created_block.location), 'courseKey': unicode(created_block.location.course_key)}
     )
 
 
+def _get_source_index(source_usage_key):
+    """
+    Get source index position of the XBlock.
```

Comment:
I think `position` is redundant.  I think `Get source index of the XBlock.` is enough?